handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/filter/IsolationServerListFilter.java

```diff
@@ -91,19 +94,20 @@ private boolean allowVisit(Server server) {
     ServerStats serverStats = stats.getSingleServerStat(server);
     long totalRequest = serverStats.getTotalRequestsCount();
     long failureRequest = serverStats.getSuccessiveConnectionFailureCount();
-
+    int currentCountinuousFailureCount = ((CseServer) server).getCountinuousFailureCount();
```

Comment:
why move calc to this place?

Reply:
currentErrorThresholdPercentage only need to be calc in one condition
but now, always do this calc.

Reply:
done. Calculate currentErrorThresholdPercentage after the conditions are met